tests/get-acey-deucey.test.js

```diff
@@ -0,0 +1,100 @@
+"use strict";
+
+const test = require("tape");
+const aceyDeuceyGameEngine = require("../");
+const getAceyDeucey = aceyDeuceyGameEngine.getAceyDeucey;
+
+test("get acey deucey", t => {
+    t.test("two dice", t => {
+        t.plan(4);
+        
+        const aceyDeucey = {isAceyDeucey: false};
+        const oneDiceRoll = [1, 6];
+        t.deepEqual(getAceyDeucey(oneDiceRoll), aceyDeucey, "returns the correct object when two dice contain a 1");
+        
+        const twoDiceRoll = [4, 2];
+        t.deepEqual(getAceyDeucey(twoDiceRoll), aceyDeucey, "returns the correct object when two dice contain a 2");
+        
+        const diceRoll = [5, 3];
+        t.deepEqual(
+            getAceyDeucey(diceRoll),
+            aceyDeucey,
+            "returns the correct object when two dice do not contain a 1 or 2"
+        );
+        const aceyDeuceyRoll = [1, 2];
+
+        try {
+            getAceyDeucey(aceyDeuceyRoll);
+            t.fail("getAceyDeucey should have thrown an error for an acey deucey roll without doubles");
+        } catch (error) {
+            t.equal(error.diceRoll, aceyDeuceyRoll, "error.diceRoll is the same as diceRoll");
```

Comment:
I would do a `deepEqual` here, since I suspect that we care that `diceRoll` has the same members as `aceyDeuceyRoll`, not `diceRoll === aceyDeuceyRoll`.